src/test/kotlin/three/consulting/epoc/repository/CustomerServiceIntegrationTest.kt

```diff
@@ -68,4 +69,19 @@ class CustomerServiceIntegrationTest : IntegrationTest() {
             .isInstanceOf(UnableToUpdateCustomerException::class.java)
             .hasMessage("Cannot update customer, missing customer id")
     }
+
+    @Test
+    fun `delete customer removes customer from the database`() {
+        assertThat(customerService.findCustomerForId(2L)).isNotNull
+        customerService.deleteCustomer(2L)
+        assertThat(customerService.findCustomerForId(2L)).isNull()
+    }
```

Comment:
Is this overly complicated or would it be enough to just run delete and assert that no errors are thrown?

Reply:
No this is perfectly fine, if you prefer to verify the existence of the customer with said id first.